exercises/run-length-encoding/tests/run-length-encoding.rs

```diff
@@ -0,0 +1,87 @@
+extern crate run_length_encoding as cipher;
```

Comment:
I'm unsure about it because I wouldn't consider run-length encoding as a means to obscure a message to prevent it from being read. That's what I think the definition of a cipher, but I apologise if I was mistaken since English isn't my first language. I'm not sure I have a better suggestion though - simply `encoding`, perhaps? Is shortening it to `rle` acceptable in Rust?